File: clients/typescript-angular/API/Client/Affiliate.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface Affiliate {

        account: number;

        currency: string;

        prevPayout?: number;

        prevTurnover?: number;

        prevComm?: number;

        prevTimestamp?: date;

        execTurnover?: number;

        execComm?: number;

        totalReferrals?: number;

        totalTurnover?: number;

        totalComm?: number;

        payoutPcnt?: number;

        pendingPayout?: number;

        timestamp?: date;
    }

}

File: clients/async-scala/src/main/scala/io/swagger/client/model/Affiliate.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Affiliate (
  account: Number,
currency: String,
prevPayout: Option[Number],
prevTurnover: Option[Number],
prevComm: Option[Number],
prevTimestamp: Option[Date],
execTurnover: Option[Number],
execComm: Option[Number],
totalReferrals: Option[Number],
totalTurnover: Option[Number],
totalComm: Option[Number],
payoutPcnt: Option[Double],
pendingPayout: Option[Number],
timestamp: Option[Date]
)
